client/analytics/report/orders/utils.js

```diff
@@ -22,7 +22,9 @@ export function formatTableOrders( orders ) {
 			total_tax,
 			shipping_total,
 			discount_total,
+			refunds,
 		} = order;
+		const refundTotal = refunds.reduce( ( acc, refund ) => acc - parseInt( refund.total ), 0 );
```

Comment:
`parseInt` rounds a decimal down here. Is that intentional? If we're displaying decimals I'd expect numbers not be rounded down (or up)

Reply:
Great catch @greenafrican - definitely did not want to do that.  While fixing this I stumbled across `getOrderRefundTotal()` which already does this the correct way with `parseFloat()` so I dropped that in 👍